File: code/6f105.pas
program Bai105;
var a, b, c: real;
x1, x2, dt: real;
begin
  readln(a,b, c);
  dt := b*b - 4*a*c;
    if dt >= 0 then
     begin
     x1 := (-b + sqrt(dt))/(2*a);
     x2 := (-b - sqrt(dt))/(2*a);
     writeln('Co 2 nghiem x1 =   ',x1:5:3 , '  nghiem x2 =   ',x2:5:3)
     end
    else  writeln('Phuong trinh vo nghiem'); 
end.

File: code/6f105.py
import math
a, b, c = [int(x) for x in input().split()]
dt = b* b - 4 *a*c
k = math.sqrt(dt)
if dt < 0:
 print('PT VO NGHIEM')
else:
  x1 = (-b + k)/2*a 
  x2 = (-b - k)/2*a
  print("%.3f" %x1,"%.3f" %x2 )